dev/table_info.h

```diff
@@ -12,6 +14,7 @@ namespace sqlite_orm {
         std::string dflt_value;
         int pk = 0;
 
+#ifndef SQLITE_ORM_AGGREGATE_PAREN_INIT
```

Comment:
what is the purpose of `SQLITE_ORM_AGGREGATE_PAREN_INIT`? It covers all trivial constructors?

Reply:
I am not sure I exactly understand what you mean by whether it covers all trivial constructors.
C++20 allows to initialize aggregates via parentheses, so e.g. `table_info()` works as well as `vector<table_info>::.emplace_back()`. So this constructor is no longer needed.

Reply:
I mean do we need to use `SQLITE_ORM_AGGREGATE_PAREN_INIT` for defining ctors in every struct?

Reply:
No, only those that will be constructed using `make_unique` or `emplace_back`, which call the parentheses constructor instead of the brace initializer. In this specific case, table_info gets emplaced into the container. Which in turn is the only one of that kind in sqlite_orm.